app/shared/text-editor/example-text-editor.tsx

```diff
@@ -0,0 +1,106 @@
+import { publish, useAllUserProps } from "@webstudio-is/sdk";
+import {
+  type EditorState,
+  LexicalComposer,
+  RichTextPlugin,
+  ContentEditable,
+  HistoryPlugin,
+  OnChangePlugin,
+} from "./lexical";
+import { InstancePlugin } from "./plugins/plugin-instance";
+import { TreeViewPlugin } from "./plugins/tree-view-plugin";
+import { config } from "./config";
+import { createInstance } from "../tree-utils";
+
+type ExampleTextEditorProps = {
+  onChange: (state: EditorState) => void;
+};
+
+const Menu = () => {
+  const [, setAllUserProps] = useAllUserProps();
+  return (
+    <div className="editor-menu">
+      <button
```

Comment:
Can our design-system components be used here?

Reply:
yep